### PR TITLE
Feat/ping

### PR DESCRIPTION
**Pull request by Ose-4g**
Created endpoint /ping to return json data telling the state of the server. 

live test endpoint = https://goals.zuri.chat/api/v1/ping

It fixes issue #357

### CODE REVIEW

**Comment by Ose-4g** [comment]
@Asylcreek  I've made a new pull request

**Comment by Asylcreek** [comment]
Hey man. Question: Are you running the server in production mode? 'Cause the log files are supposed to be empty.

And please change the "api/v1/ping" to "/ping", this is because the frontend is not consuming this route.

And please pull from the staging branch before your next push, there are some updates your code doesn't have.

**Comment by Ose-4g** [comment]
Alright @Asylcreek I've made the changes. 
- Removed all logs from the error files
- Changed my NODE_ENV to development locally
- Changed the endpoint to /ping
- Pulled updates from upstream staging to avoid merge conflicts

File: app.js [hunk 1]
```diff
@@ -46,6 +47,9 @@ app.use(compression());
 
 // Api routes
 app.use('/api/v1/goals', rateLimiter(), goalRouter);
+app.use('/ping', rateLimiter(), pingRouter);
+app.use('/api/v1/info', rateLimiter(), pluginInfoRouter);
```
Comment (Asylcreek) [inline app.js:51]: Please delete this line

File: app.js [hunk 2]
```diff
@@ -46,6 +47,9 @@ app.use(compression());
...
+app.use('/api/v1/sidebar', rateLimiter(), sidebarRouter);
```
Comment (Asylcreek) [inline app.js:52]: Please delete this line

**Review by Asylcreek — CHANGES_REQUESTED** [review]

File: app.js (hunk 2 above)
Comment (Ose-4g) [reply app.js:52]: Alright. So I shouldn't use a router for the endpoint ???

**Comment by Ose-4g** [comment]
Alright @Asylcreek  I've updated the document as requuested.

**Pull request merged**